api/tests/unit/infrastructure/serializers/csv/campaign-profile-collection-export_test.js

```diff
@@ -106,7 +106,7 @@ describe('Unit | Serializer | CSV | campaign-profiles-collection-export', functi
       expect(csv).to.equal(expectedHeader);
     });
 
-    it('should display student number header when organization is SUP and managing students', async function() {
+    it('should display both student number header and group header when organization is SUP and managing students', async function() {
```

Comment:
**suggestion (non-blocking):** `It displays the headers for SUP organization which manages students`
Comme ça si on change encore des colonnes la description reste correcte ?

Reply:
je suis d’accord avec toi.